src/main/java/core/basesyntax/service/OperationTypeService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationTypeService {
```

Comment:
delete this interface. 
Simple add method `getByCode(String code)` to enum Operation